### ISSUE TITLE
report function in API not working

### ISSUE THREAD

**Issue by mesmith75** [ganga-devs/ganga#1023]
The report function is giving an error:
```python
Ganga In [16]: report(jobs(1916))
ERROR    No module named config
INFO     job 1918 status changed to "running"
ERROR    !!Unknown/Unexpected ERROR!!
ERROR    If you're able to reproduce this please report this to the Ganga developers!
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/afs/cern.ch/user/m/masmith/cmtuser/GANGA/GANGA_HEAD/install/ganga/python/Ganga/GPI/__init__.pyc in <module>()
----> 1 report(jobs(1916))

/afs/cern.ch/user/m/masmith/cmtuser/GANGA/GANGA_HEAD/install/ganga/python/Ganga/Utility/feedback_report.pyc in report(job)
     37     import platform
     38 
---> 39     import Ganga.GPIDev.Lib.Config.config as config
     40     from Ganga.GPIDev.Base.VPrinter import full_print
     41 

ImportError: No module named config
```

**Comment by rob-c** [comment]
@mesmith75 Give me a nudge when I'm next on dast shift and I'll look at bringing this functionality back and restoring the report server.
(Or can we upload this stuff to a cernbox account now and avoid having to host another service for this)

**Comment by egede** [comment]
To write into a cernbox directory, I think would require authentication against the account. Not sure we could do that in any way without violating CERN guidelines. If a way can be found around that, it is a much better idea though.

**Comment by rob-c** [comment]
@egede I wonder are we allowed CERN-Box accounts for the service accounts and they could share a public folder using the cernbox sharing authentication. If this allows uploading files to a folder but only for people who have the shared credentials (which we would embed in ganga) then this sounds safe enough. If there is no way to share a folder so a non-CERN account can write to it then maybe this isn't the best solution. I don't know how fine grained the permission settings are
Atm the old protocol allowed for anyone with acces to the ganga codebase to upload anything under 10Mb to the old server which may run afoul of a few security best practices. I don't

**Comment by egede** [comment]
The `report` feature is used as a way for a user to upload their configuration and then get help with debugging an issue. The current implementation has not been maintained and some issues should be solved:

- The import error should be fixed. A better way is to do `import GangaCore.GPIDev.Lib.Config` and then just refer to it as `GangaCore.GPIDev.Lib.Config.config`
- The server that the report used to be uploaded to is no longer available. Investigate some public service instead. Would unlisted guest posts to `Pastebin` work?
- Write some unit tests to make sure that function does not go out of use again.
- Update user manual to advertise availability.

**Comment by RajatSingh95** [comment]
@egede which file has the server configuration where reports has to be uploaded?

**Comment by egede** [comment]
In your home directory there is a hidden configuration file called .gangarc. In that one you will find

```
#  Settings for the Feedback plugin. Cannot be changed during the interactive
#  Ganga session.
[Feedback]

#  The server to connect to
#uploadServer = http://gangamon.cern.ch/django/errorreports
```
If you take away the '#' at the beginning of the line you can change the default value to anything else.

Another (and eventually permanent) way is to change the default value which is set in the file `ganga/GangaCore/__init__.py`

**Comment by RajatSingh95** [comment]
@egede I have made changes to `feedback_report.py`  and tested the report function on a local django server by creating a service which accepts the request from Ganga.
I have added commits regarding the change. Please have a look.
![report_output](https://user-images.githubusercontent.com/26798329/75267348-24537800-581c-11ea-9a65-3b1b3366f034.png)

**Comment by egede** [comment]
Fixed by #1587
